koku/api/report/aws/query_handler.py

```diff
@@ -198,6 +198,13 @@ def execute_query(self):
         data = []
 
         with tenant_context(self.tenant):
+
+            if not self.parameters.parameters.get('compute_count'):
+                # Query parameter indicates count should be removed from DB queries
+                self._mapper.report_type_map.get('aggregates').pop('count', None)
```

Comment:
The mapper object should be treated as read-only.

Instead of `.pop()`, can you use `.get()`?

Reply:
Lot's of chat offline.  I migrated to what we chatted about.  Essentially using a deepcopy and not changing the maps.